datacapturing/src/cyface/java/de/cyface/datacapturing/CyfaceDataCapturingService.java

```diff
@@ -190,4 +192,42 @@ public void start(@NonNull Vehicle vehicle, @NonNull StartUpFinishedHandler fini
             }
         }
     }
+
+    /**
+     * Sets up an already existing {@code Account} to work with the {@link WiFiSurveyor}.
+     * <p>
+     * <b>ATTENTION:</b> SDK implementing apps need to use this method if they cannot use
+     * {@link WiFiSurveyor#createAccount(String, String)}.
+     * <p>
+     * This has the following reasons:
+     * - {@code ContentResolver#addPeriodicSync()} is always registered until {@link WiFiSurveyor#deleteAccount(String)}
+     * is called
+     * - {@code ContentResolver#setSyncAutomatically()} is automatically updated via {@link NetworkCallback}s and
+     * defines if a connection is available which can be used for synchronization (dependent on
+     * {@link WiFiSurveyor#setSyncOnUnMeteredNetworkOnly(boolean)}). Using this instead of the periodicSync flag fixed
+     * MOV-535.
+     * - {@code ContentResolver#setIsSyncable()} is used to disable synchronization manually and completely
+     *
+     * @param account The {@code Account} to be used for synchronization
+     * @param enabled True if sync should be enabled
+     */
+    public void makeAccountSyncable(@NonNull final Account account, boolean enabled) {
+        getWiFiSurveyor().makeAccountSyncable(account, enabled);
```

Comment:
Is there a reason you are wrapping these calls to the WifiSurveyor in the DataCapturingService?

Reply:
This added control over which public WifiSurveyor methods are used but as we plan to decouple the WifiSurveyor from the DCS anyway I'll remove the wrapper which forces us to keep the WifiSurveyor APIs clean.